symphony-bdk-legacy/symphony-api-client-java/src/main/java/model/OutboundMessage.java

```diff
@@ -23,37 +24,25 @@ public OutboundMessage(String message, String data) {
         this.data = data;
     }
 
-    public OutboundMessage(String message, File[] attachment) {
+    public OutboundMessage(String message, File... attachment) {
```

Comment:
What happen if we pass a `null` value here?

Reply:
We have to cast it to File class, then the attachment array will contain only 1 null element.